bindings/java/native/src/classes/account/acc.rs

```diff
@@ -242,4 +240,49 @@ impl Account {
     pub fn last_synced_at(&self) -> Option<DateTime<Local>> {
         crate::block_on(async move { self.handle.last_synced_at().await })
     }
+
+    pub fn to_string(&self) -> String {
+        format!("{:?}", self.handle)
+    }
+}
+
+impl core::fmt::Display for Account {
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> std::fmt::Result {
+        write!(
+            f, "{:?}", self.handle
```

Comment:
I don't think this is good -- just use `Debug` if someone wants to format a debug string.

Reply:
I use the Display methods to call from the toString() through java

Reply:
As I see at line 244 you also have a `pub fn to_string` that formats to `{:?}`, which uses `Debug`. Is the Java `toString` bound to that function or to `Display`?

Reply:
As I see at line 244 you also have a `pub fn to_string` that formats to `{:?}`, which uses `Debug`. Is the Java `toString` bound to that function or to `Display`?

Reply:
It just needs to be linked to another function (example: https://github.com/iotaledger/wallet.rs/blob/93a3318c1d3f952348610051b821c75b1237550a/bindings/java/native/src/java_glue.rs.in#L152)